src/CMakeLists.txt

```diff
@@ -54,6 +54,14 @@ set(DEP_LIBS_PKG_ARG_LISTS "IpcShm ${DEP_IPC_SHM_VERSION} CONFIG")
 # (TODO: but look into the jemalloc/jemalloc-cmake GitHub repo -- what's that all about?)
 # Hence we must ensure 2 things more manually: the lib location and the include-path.
 
+# Try pkg-config if available
+pkg_check_modules(JEMALLOC REQUIRED jemalloc)
+if(JEMALLOC_FOUND)
+  set(JEMALLOC_LIB "${JEMALLOC_LIBRARIES}")
+  set(JEMALLOC_INCLUDE_PATH "${JEMALLOC_INCLUDEDIR}")
```

Comment:
Should these lines set them as CACHE variables? I ask, because find_library() and find_path() do so (unless one sets NO_CACHE, which I did not).

Reply:
> PS: Actually jemalloc releated cmake can be further simplified, current modifications are minimized to keep consistent with the existing approach.

I am not sure, they are just aliases to avoid more modifications.

Personally I recommend use `JEMALLOC_LIBRARIES` and `JEMALLOC_INCLUDEDIR` directly rather than shimming via `JEMALLOC_LIB` and `JEMALLOC_INCLUDE_PATH`:

https://cmake.org/cmake/help/latest/module/FindPkgConfig.html#command:pkg_check_modules says that `<XXX>_LIBRARIES` and `<XXX>_INCLUDE_DIRS` are set by `pkg_check_modules()`. I am not quite sure whether `find_package()` use these names too or not, but as far as I remember, manually specifying other libraries usually requires `-D <LIBNAME>_LIBRARIES=PATH_TO_LIB_DIR`.